README.md

```diff
@@ -4,14 +4,49 @@
 [![Build Status](https://github.com/software-gardening/almanack/actions/workflows/pytest-tests.yml/badge.svg?branch=main)](https://github.com/software-gardening/almanack/actions/workflows/pytest-tests.yml?query=branch%3Amain)
 ![Coverage Status](https://raw.githubusercontent.com/software-gardening/almanack/main/media/coverage-badge.svg)
 
-An open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+The Software Gardening Almanack is an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+The Almanack is available both as a book and a Python package.
 
-Please see the following links to access published content:
+## Handbook
 
 - __Online (HTML)__: https://software-gardening.github.io/almanack/
 - __Offline (PDF)__: [software-gardening-almanack.pdf](https://software-gardening.github.io/almanack/software-gardening-almanack.pdf)
 
-## CONTRIBUTING
+## Package
+
+You can install the Almanack with the following:
+
+```bash
+# install from pypi
+pip install almanack
+
+# install directly from source
+pip install git+https://github.com/software-gardening/almanack.git
+```
+
+Once installed, the Almanack may be used to analyze repositories for sustainable development practices.
+Output from the Almanack includes metrics which are defined through `metrics.yml` as a Python dictionary (JSON-compatible) record structure.
+
+For example:
+
+```python
+import almanack
+import pandas as pd
+
+# gather the almanack table using the almanack repo as a reference
+almanack_table = almanack.metrics.data.get_table("repository directory")
+
+# show the almanack table as a Pandas DataFrame
+pd.DataFrame(almanack_table)
+```
+
+Alternatively, you can use the Almanack package as a command-line interface (CLI):
+
+```bash
+almanack "repository directory"
```

Comment:
Same as above

Reply:
Thanks, will add a commit to this effect.